__tests__/get.test.js

```diff
@@ -61,6 +61,145 @@ describe("GET /api/reviews", () => {
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews.length).toBeGreaterThan(0);
+            expect(result.body.reviews[0]).toMatchObject({
+              owner: "philippaclaire9",
+              title: expect.any(String),
+              review_id: 2,
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+      });
+    });
+    describe("order", () => {
+      test("default order", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews.length).toBeGreaterThan(0);
+            expect(result.body.reviews[0]).toMatchObject({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: 13,
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              review_body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(String),
+            });
+          });
+      });
+      test("should sort in ascending order", () => {
+        return request(app)
+          .get("/api/reviews?order=asc")
+          .expect(200)
+          .then((result) => {
+            expect(result.body.reviews.length).toBeGreaterThan(0);
+            expect(result.body.reviews).toBeSortedBy("review_id");
+          });
+      });
+    });
+  });
+  describe("errors", () => {
+    describe("categories", () => {
+      test("return 400 when category provided doesnt exist", () => {
+        return request(app)
+          .get("/api/reviews?category=aubergine")
+          .expect(400)
+          .then((result) => {
+            expect(result.body.message).toBe("Category not found");
+          });
+      });
+      // Not valid test not required as it will be stringified anyway, and thus will always be valid
+    });
+    describe("sort by", () => {
+      test("400: sort by invalid column name", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=aubergine")
+          .expect(400)
+          .then((result) => {
+            expect(result.body.message).toBe("invalid query");
+          });
+      });
+    });
+    describe("400 order by", () => {
+      test("400 order by anything by asc or desc", () => {
+        return request(app)
+          .get("/api/reviews?order=aubergine")
+          .expect(400)
+          .then((result) => {
+            expect(result.body.message).toBe("invalid query");
+          });
+      });
+    });
```

Comment:
You also need to test that sort_by and order work in tandem together (e.g. /api/reviews?sort_by=owner&order=asc)